packages/altive_lints/test/lints/avoid_hardcoded_japanese.dart

```diff
@@ -1,12 +1,9 @@
 // Check the `avoid_hardcoded_japanese` rule.
 
-// expect_lint: avoid_hardcoded_japanese
```

Comment:
![badge](https://img.shields.io/badge/review-ask-informational)
Why did you remove `expect_lint`?

Reply:
The change from lint_test to test has been made so that contents in the test directory cannot warn.
This section was also removed because it is no longer possible to warn.